src/server/api/schema.js

```diff
@@ -244,7 +245,7 @@ async function editCampaign(id, campaign, loaders, user) {
       payload: compressedString.toString('base64')
     })
     if (JOBS_SAME_PROCESS) {
-      uploadContacts(job).then()
+      uploadContacts(job)
```

Comment:
we need to keep these `.then()`s same as below, at least for prod-ish environments.

Were you intending to complete this as:
```
if (JOBS_SAME_PROCESS) {
   (JOBS_SYNC ? await uploadContacts(job) : uploadContacts(job).then())
}
```

Reply:
I believe these `.then()` calls aren't doing anything useful. I think they're only cloning the promise object returned by the async function, without changing it at all.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then is the closest thing I could find to directly describing an empty `.then()` call:

> If one or both arguments are omitted, or are provided non-functions, then then will be missing the handler(s), but will not generate any errors.